cmd/kpod/inspect.go

```diff
@@ -83,7 +83,10 @@ func inspectCmd(c *cli.Context) error {
 	}
 	server, err := libkpod.New(config)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "could not get container server")
+	}
+	if err = server.Update(); err != nil {
```

Comment:
Hm. Fair. We should be doing it for every command, though - maybe make a common function for every kpod command creates the ContainerServer and runs Update()?